src/main/java/com/google/api/generator/gapic/protoparser/Parser.java

```diff
@@ -709,6 +710,9 @@ private static boolean isMapType(Descriptor messageDescriptor) {
                   .getOptions()
                   .getExtension(ExtendedOperationsProto.operationPollingMethod)
               : false;
+      // may not need to pass in messageTypes?
```

Comment:
I don't understand what this is saying and why. Remove or resolve it. Or add an unambiguous TODO.

Reply:
Removed

Reply:
Make sure you push your changes before making this as resolved.

Reply:
Changes pushed.